src/main/streamer.ts

```diff
@@ -122,6 +122,8 @@ function computeReadiumCssJsonMessage(publication: Publication, link: Link | und
         typeScale: readiumCSSDefaults.typeScale,
 
         wordSpacing: settings.wordSpacing,
+
+        reduceMotion: false,
```

Comment:
I suggest to use `readiumCSSDefaults.reduceMotion`, as in the original Pull Request: https://github.com/readium/readium-desktop/pull/202

Reply:
Fixed in this commit: https://github.com/readium/readium-desktop/pull/212/commits/74cbb6be8232b8d41e360efd72d2c4cec05a2439